### PR TITLE
5 diff view

### LINKED ISSUE #5: Add diff view in code

### PR DESCRIPTION
**Pull request by spaenleh**
This PR adds the diff view to the code capsule app.

Diff View in Player
Screenshot 2022-11-16 at 19 30 26

Diff View settings in Builder
Screenshot 2022-11-16 at 21 15 23

Diff view Preview in settings
Screenshot 2022-11-16 at 21 15 30

closes #5

### CODE REVIEW

File: src/components/views/read/PlayerView.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { CodeVersionProvider } from '../../context/CodeVersionContext';
 import { SettingsProvider } from '../../context/SettingsContext';
 
 const Div = styled('div')({
-  height: '100vh',
+  // height: '100vh',
 });
```
Comment (pyphilia) [inline src/components/views/read/PlayerView.tsx:12]: Remove?

**Review by pyphilia — APPROVED** [review]
LGTM 🍔

File: package.json [hunk 2]
```diff
@@ -50,6 +50,7 @@
     "prism-react-renderer": "1.3.5",
     "qs": "6.11.0",
     "react": "18.2.0",
+    "react-diff-viewer": "^3.1.1",
```
Comment (juancarlosfarah) [inline package.json:53]: Fix version?

**Review by juancarlosfarah — APPROVED** [review]
👨‍💻

File: package.json (hunk 2 above)
Comment (juancarlosfarah) [reply package.json:53]: @spaenleh not an important change, but something to keep in mind.

**Pull request merged**